desktop/src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java

```diff
@@ -373,6 +375,12 @@ private void showPublishTxPopup(Coin receiverAmount,
                         @Override
                         public void onSuccess(Transaction transaction) {
                             log.debug("Successfully sent tx with id {}", txWithBtcFee.getTxId().toString());
+                            String key = "showTransactionSentBsq";
```

Comment:
It would be great to get rid of the error message on the empty fields after the successful transfer as well.
For the `BsqSendView` this would mean changing in the `showPublishTxPopup` resultHandler into
```
() -> {
       receiversAddressInputTextField.setValidator(null);
       receiversAddressInputTextField.setText("");
       receiversAddressInputTextField.setValidator(bsqAddressValidator);
       amountInputTextField.setValidator(null);
       amountInputTextField.setText("");
       amountInputTextField.setValidator(bsqValidator);
       });
```

Reply:
Not beautiful, but it works :wink:

Reply:
Thanks @ripcurlx, implemented your suggestions. :+1: